### ISSUE TITLE
Does identation lines works in v.5.0.1?

### ISSUE THREAD

**Issue by JohannHM** [spyder-ide/spyder-terminal#256]
## Description of your problem
Identation vertical lines (guides) don't display in v5.0.1, even if identation guides are marked 
 
**What steps will reproduce the problem?**

1. identation guides are marked but lines don't  show

**What is the expected output? What do you see instead?**
identation vertical lines

**Please provide any additional information below**

**Comment by steff456** [comment]
Hi @JohannHM,

This is an issue you are having with Spyder or Spyder-terminal? Did Spyder throw an error message or the functionality is just not working? Either way please upload a screenshot with your behavior

Thanks for reporting

**Comment by JohannHM** [comment]
Hi team.
It's an issue with spyder.
spyder 5 does not show the vertical lines for better indentation. After @ccordoba12 recommendation of downgrading the version the solution was still worst. All the envs worked wrong, screens doesn't fit, resolution crashed off, I had to completely remove spyder spending so many time to go back to the basic configuration. No pics available now, cuz I had to re-install all again back.
